pkg/extension/sumologicextension/config.go

```diff
@@ -91,6 +91,11 @@ type Config struct {
 	// Exponential algorithm is being used.
 	// Please see following link for details: https://github.com/cenkalti/backoff
 	BackOff backOffConfig `mapstructure:"backoff"`
+
+	// IsRemotelyManaged field defines management status of the OT Collector based on
+	// if it is remotely or locally managed
+	// By default this is false.
+	IsRemotelyManaged bool `mapstructure:"isRemotelyManaged"`
```

Comment:
```suggestion
	IsRemotelyManaged bool `mapstructure:"is_remotely_managed"`
```
Otel configurations use snake case by convention.